lib/facade_converter.ts

```diff
@@ -244,6 +244,16 @@ export class FacadeConverter extends base.TranspilerBase {
         this.emitCall('remove', c.arguments);
         this.emit('!= null || true ) )');
       },
+      'Map.forEach': (c: ts.CallExpression, context: ts.Expression) => {
+        const callback = <ts.FunctionExpression>c.arguments[0];
+        let params = callback.parameters;
+        if (params.length !== 2) {
+          this.reportError(c, 'Map.forEach callback supports two arguments only');
+        }
+        [params[0], params[1]] = [params[1], params[0]];
```

Comment:
@mprobst I would prefer creating a new Node instead of mutating an existing one. What do you think ?